test/service/test_response.py

```diff
@@ -1393,24 +1393,51 @@ def lambda_name(cls) -> str:
 
     maxDiff = None
 
+    def _get_integrations(self, params=None):
+        url = self.base_url + '/integrations'
+        response = requests.get(url, params=params)
+        response.raise_for_status()
+        return response.json()
+
+    # TODO: Change test to be more generic, not specific to current contents of portal integrations db
```

Comment:
Either resolve to do right away or file a ticket and mention here. Having said that, see below.

Reply:
Fixing the implementation by using mocked data